app/helpers/embed_player_helper.rb

```diff
@@ -18,6 +18,10 @@ module EmbedPlayerHelper
   EMBED_PLAYER_SEASON = "se"
   EMBED_PLAYER_CATEGORY = "ct"
 
+  def embed_url?(value)
+    value.blank? || value.include?(ENV["PLAY_HOST"])
```

Comment:
Maybe this `include?` is over-simplistic?  But hard to imagine a different and real url would have `play.prx.org` in it somewhere.